tuned/hardware/inventory.py

```diff
@@ -25,7 +25,11 @@ def __init__(self, udev_context=None, udev_monitor_cls=None, monitor_observer_fa
 			buffer_size = consts.CFG_DEF_UDEV_BUFFER_SIZE
 
 		if (set_receive_buffer_size):
-			self._udev_monitor.set_receive_buffer_size(buffer_size)
+			try:
+				self._udev_monitor.set_receive_buffer_size(buffer_size)
+			except EnvironmentError:
+				log.warn("cannot set udev monitor receive buffer size, we are probably running inside " + \
```

Comment:
The backslash at the end of the line is unnecessary because it's between parentheses.

Reply:
Sorry, the plus is unnecessary too. But don't worry about it :D.

Reply:
Do you want me to remove it? Personally, I don't care, but maybe with the "+" it seems more readable to me. IMHO similar constructs are already in the code.

Reply:
OK, we can leave it.